src/Mono.Android/Android.Runtime/Logger.cs

```diff
@@ -58,18 +60,26 @@ internal static class Logger {
 					System.Console.WriteLine ("[{0}] {1}: {2}", level, appname, line);
 			}
 		}
+
+		[DllImport ("__Internal", CallingConvention = CallingConvention.Cdecl)]
+		extern static uint monodroid_get_log_categories ();
+
+		static Logger ()
+		{
+			logCategories = (LogCategories) monodroid_get_log_categories ();
+		}
 	}
 
 	internal enum LogLevel {
```

Comment:
If these need to be kept in-sync with `libmonodroid`, we should add an appropriate comment here, as was done with `LogCategories`.